x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RequestHandler } from 'kibana/server';
+import {
+  GetTrustedAppsListRequest,
+  GetTrustedListAppsResponse,
+} from '../../../../common/endpoint/types';
+import { EndpointAppContext } from '../../types';
+import { exceptionItemToTrustedAppItem } from './utils';
+
+export const getTrustedAppsListRouteHandler = (
+  endpointAppContext: EndpointAppContext
+): RequestHandler<undefined, GetTrustedAppsListRequest> => {
+  const exceptionsListService = endpointAppContext.service.getExceptionsList();
+  const logger = endpointAppContext.logFactory.get('trusted_apps');
+
+  return async (context, req, res) => {
+    const { page, per_page: perPage } = req.query;
```

Comment:
This is why I'm wondering if it's a convention to follow "_" naming style for REST :)